auto/help

```diff
@@ -76,7 +76,12 @@ cat << END
   java OPTIONS         configure Java module
                        run "./configure java --help" to see available options
 
-  wasm OPTIONS	       configure WebAssembly module
-		       run "./configure wasm --help" to see available options
+  wasm OPTIONS         configure WebAssembly module
```

Comment:
99% of our users will want `wasm-wasi-component`... I wonder if we can signpost this better?

```suggestion
  wasm OPTIONS         configure raw (not Component Model) WebAssembly module
```

Hm. I don't like that either. Food for thought if you have something better that immediately comes to mind.

Not blocking on this feedback.